end2end-test-examples/echo-client/src/main/java/io/grpc/echo/EchoClient.java

```diff
@@ -174,13 +199,19 @@ void blockingEcho(Histogram histogram) {
       }
       long cost = System.currentTimeMillis() - start;
       if (histogram != null) histogram.recordValue(cost);
+      logger.info(String.format("RPC succeeded after %d ms.", cost));
```

Comment:
same comment as prev log info statement

Reply:
You are right. Here if we send many requests without any delay it won't be very helpful. Updated to only log when we send infinite requests with some delay because we want to know how long it took for each request to complete.